javaparser-testing/src/test/resources/com/github/javaparser/storage/PrimaryType.java

```diff
@@ -0,0 +1,5 @@
+package com.github.javaparser.storage;
+
+public class PrimaryType {
+    
+}
```

Comment:
Newline :)

This is my favorite type of comments :)

Reply:
This PR allows edits from maintainers, so if you consider this so important.... ;-)